benchmarks/fasta/php/bench.php

```diff
@@ -10,7 +10,8 @@
 ob_implicit_flush(1);
 ob_start(NULL, 4096);
 
-$last = 42.0;
+$INITIAL_STATE = 42.0;
```

Comment:
From memory, using `define` is faster in Zend (and probably Hippy, a bit). I might be wrong.